share/scripts/inverse/run_gromacs_multidir.sh

```diff
@@ -0,0 +1,342 @@
+
+[[ -n ${mdirs} ]] || die "${0##*/}: mdrun '-multidir' option does not contain directory names including pattern 'sim' (check cg.inverse.gromacs.mdrun.opts)"
+[[ ${mdirs} -gt 1 ]] || die "${0##*/}: mdrun '-multidir' option is set but the number of directories including pattern 'sim' <= 1 (check cg.inverse.gromacs.mdrun.opts)"
+[[ ${mdirs} -le ${tasks} ]] || die "${0##*/}: mdrun '-multidir' option provided presumes the number of separate simulations greater than the number of tasks/threads (check cg.inverse.gromacs.mdrun.opts)"
+
+echo -e "\nCreating/checking simulation data for ${mdirs} separate simulations (run in parallel)"
+
+confout="$(csg_get_property cg.inverse.gromacs.conf_out)"
+conf="$(csg_get_property cg.inverse.gromacs.conf)"
+mdp="$(csg_get_property cg.inverse.gromacs.mdp)"
+index="$(csg_get_property cg.inverse.gromacs.index)"
+topol_in="$(csg_get_property cg.inverse.gromacs.topol_in)"
+traj="$(csg_get_property cg.inverse.gromacs.traj)"
+checkpoint="$(csg_get_property cg.inverse.gromacs.mdrun.checkpoint)"
+tpr="$(csg_get_property cg.inverse.gromacs.topol)"
+
+grompp_opts="$(csg_get_property --allow-empty cg.inverse.gromacs.grompp.opts)"
+grompp="$(csg_get_property cg.inverse.gromacs.grompp.bin)"
+[[ -n "$(type -p ${grompp})" ]] && echo "using grompp binary '${grompp}'" || die "${0##*/}: grompp binary '${grompp}' not found (override by cg.inverse.gromacs.grompp.bin)"
+
+if is_done "Simtasks"; then
+    echo -e "\nAll Simulation tasks done - skipping\n"
+else
+    for ((it=0;it<mdirs;it++)); do
+	this_dir="sim_${it}"
+
+	if [[ -d $this_dir ]]; then
+	    cd $this_dir || die "${0##*/}: cd $this_dir failed"
+	    if is_done "Simulation"; then 
+		echo -e "\nSimulation task ${it} done - skipping\n"
+		cd ../
+		continue; 
+	    fi
+	    cd ../
+	else
+	    echo -e "\nSimulation task $it created at $(date)\n"
+	    mkdir -p $this_dir || die "${0##*/}: mkdir -p $this_dir failed (unfinished simulation exists?)"
+	fi
+
+	cd $this_dir || die "${0##*/}: cd $this_dir failed"
+
+	filelist="$(csg_get_property --allow-empty cg.inverse.filelist)"
+
+	if is_done "Filecopy"; then
+        #check for needed files 
+	    echo "Filecopy already done"
+	    for f in $filelist; do
+
+		[[ -f $f ]] || cp_from_main_dir "$f"
+
+		echo Comparing "$(get_main_dir)/$f" "$f"
+		[[ -z $(type -p cmp) ]] && echo "${0##*/}: program 'cmp' not found, comparision skipped" && continue
+
+		cmp "$(get_main_dir)/$f" "$f" && echo "Unchanged" || \
+		msg --color blue --to-stderr "${0##*/}: WARNING: file '$f' in the main dir was changed since the last execution, this will have no effect on the current iteration, to take effect remove the current iteration ('${this_dir##*/}')"
+
+	    done
+	else
+        #get needed files 
+	    echo "Copying the needed files"
+	    [[ -n ${filelist} ]] && cp_from_main_dir "$filelist"
+
+	    mark_done "Filecopy"
+	fi
+	
+	if [[ -f ${conf}.${it} ]]; then
+    	   critical cp -f "${conf}.${it}" "$conf"
```

Comment:
According to the latest documentation (http://manual.gromacs.org/documentation/2016.3/user-guide/mdrun-features.html), the main difference between -multidir and -multi is that in the former case separate simulation inputs/outputs are dealt with in separate directories vs in the latter case all files are dealt with from within the same directory but files recognised by the numeric suffix preceding the extension dot. 

I opted for the sub-directories (now I recall it!) as it is cleaner to check and I could abuse existing mechanisms in VOTCA for capturing errors and dealing with restarts etc (as all the names in a subdir are the same as for a single simulation. I admit, it is sort of non-elegant to copy and rename the initially suffixed input files but at least it is done in only one additional script (i.e. minimising my intervention into the existing VOTCA scripts).

With the '-multi' option everything would go in the same dir... run_gromacs.sh and possibly other gromacs-related scripts would need to be amended.